src/Tribe/Support.php

```diff
@@ -44,10 +44,14 @@ private function __construct() {
 			$this->must_escape = (array) apply_filters( 'tribe_help_must_escape_fields', $this->must_escape );
 			add_action( 'tribe_help_pre_get_sections', array( $this, 'append_system_info' ), 10 );
 			add_action( 'delete_option_rewrite_rules', array( $this, 'log_rewrite_rule_purge' ) );
+
+			add_action( 'rest_api_init', array( $this, 'create_sysinfo_endpoint' ) );
+			add_action( 'wp_ajax_tribe_toggle_sysinfo_optin', [ $this, 'ajax_sysinfo_optin' ] );
```

Comment:
Ditto here - `ajax_sysinfo_optin()` is static so we should change how we call it (alternatively, perhaps those methods shouldn't be static?).